### PR TITLE
1848 rename whole dataset drift check to multivariate drift

### LINKED ISSUE #1848: [FEAT] Rename Whole Dataset Drift check to Multivariate Drift
**Is your feature request related to a problem? Please describe.**
Multivariable drift is a better name than the WholeDatasetDrift. Several requests from the community came in about that.

### PR DESCRIPTION
**Pull request by kishore-s-15**
Closes #1848

### CODE REVIEW

File: deepchecks/checks.py [hunk 1]
```diff
@@ -62,7 +62,7 @@
     # distribution checks
     'TrainTestFeatureDrift',
     'TrainTestLabelDrift',
-    'WholeDatasetDrift',
+    'MultiVariateDrift',
```
Comment (ItayGabbay) [inline deepchecks/checks.py:65]: I think the name (in all changes) should be `MultivariateDrift` instead of `MultiVariateDrift` (mind the V)

**Review by ItayGabbay — COMMENTED** [review]
Great job! Added one comment.

File: deepchecks/checks.py (hunk 1 above)
Comment (kishore-s-15) [reply deepchecks/checks.py:65]: Sure, I'll make the necessary changes.

File: deepchecks/tabular/checks/train_test_validation/multivariate_drift.py [hunk 2]
```diff
@@ -12,14 +12,14 @@
...
+from deepchecks.core.check_utils.multivariate_drift_utils import run_multivariate_drift
 from deepchecks.tabular import Context, TrainTestCheck
 from deepchecks.utils.strings import format_number
 
-__all__ = ['WholeDatasetDrift']
+__all__ = ['MultivariateDrift']
 
 
-class WholeDatasetDrift(TrainTestCheck):
+class MultivariateDrift(TrainTestCheck):
```
Comment (Nadav-Barak) [inline deepchecks/tabular/checks/train_test_validation/multivariate_drift.py:22]: Would add a deprecation warning to the old name and reference to new check name, can see an example for that in: deepchecks/tabular/checks/model_evaluation/performance_report.py

Comment (kishore-s-15) [reply deepchecks/tabular/checks/train_test_validation/multivariate_drift.py:22]: Should I add the warning message on the `__init__` method of `MultivariateDrift` class or similar to `PerformanceReport` from deepchecks/tabular/checks/model_evaluation/performance_report.py ?

Comment (kishore-s-15) [reply deepchecks/tabular/checks/train_test_validation/multivariate_drift.py:22]: @Nadav-Barak I have made the requested change, could you check if that's fine ?

**Comment by kishore-s-15** [comment]
@ItayGabbay @Nadav-Barak @shir22 Hi Guys, Any updates on this PR 🙂 ?

**Comment by kishore-s-15** [comment]
> @kishore-s-15 Hey! After we noticed there are some users who are using the ImageDatasetDrift, I think we shouldn't remove it. In addition, after reviewing your PR I suggest that we should support the old name (WholeDatasetDrift) but with a deprecation warning. Exactly like we did in the SegmentPerformance check.

I'll make the necessary changes.

**Comment by shir22** [comment]
Hi @kishore-s-15 !

For removal you can specify version number 0.11.
Deprecated - since the current version of when this PR will be merged, which depends... (the next version to be released is 0.9, and it will be released this week).

By the way - notice that PR's like this that touch multiple existing files  that are many times being updated, it would be best to try finalize it in one shot (with quick fixes so that we can have it merged into main in a day or so) otherwise we'll have to keep merging from main and dealing manually with the new conflicts (like currently - there are many changes that have to be updated, it might even be easiest working on a new completely pull from main..).
If you prefer that someone else assist you in finalizing this issue due to the nature of this PR, feel free to state so, we'll be happy to do it.

One small note for best practice - make sure to "rename" the renamed files using git (so that history is tracked), instead of rename/move them using your operating system, cause then git will consider that a deleted + new file instead.

**Comment by kishore-s-15** [comment]
Sure, @shir22. As mentioned, I'll specify the deprecation version as 0.9 & removal version as 0.11. 

So far in this PR, I have removed `ImageDatasetDrift` check & renamed the `WholeDatasetDrift` (in a lot of files), which are not necessary now. I'll close this PR and start working on a new branch from the latest `main` branch. 

I never knew `git mv` existed previously, I've always renamed the files using the operation system. Thanks for letting me know of this feature, it'll be really helpful for me. I'll rename files using it in the future.

**Comment by kishore-s-15** [comment]
Moved to https://github.com/deepchecks/deepchecks/pull/2011

**Pull request closed without merging**